bot/tests/fixtures.py

```diff
@@ -0,0 +1,21 @@
+from typing import Optional
```

Comment:
I think we might want to move all the test stuff out of `kodiak/` eventually to make it easier to understanding what is for testing.